src/app/Main/customer-about-us/customer-about-us.component.html

```diff
@@ -1 +1,27 @@
-<p>customer-about-us works!</p>
+<div class="" >
```

Comment:
Try to develop the code inside <div class="main-content> <div class="container-fluid> </div> </div>
@ThiliniUdari  Upload a screenshot of the components you have designed

Reply:
![Screenshot_2020-05-17 Material Dashboard Angular by Creative Tim(1)](https://user-images.githubusercontent.com/43146276/82155485-e9993300-9892-11ea-8382-54d3427a5c36.jpg)
![Screenshot_2020-05-17 Material Dashboard Angular by Creative Tim](https://user-images.githubusercontent.com/43146276/82155494-f1f16e00-9892-11ea-835b-6ba96a06656e.png)
![Screenshot_2020-05-17 Material Dashboard Angular by Creative Tim(1)](https://user-images.githubusercontent.com/43146276/82155498-f74eb880-9892-11ea-9df7-67915e70871a.png)
![Screenshot_2020-05-17 Material Dashboard Angular by Creative Tim(2)](https://user-images.githubusercontent.com/43146276/82155504-fddd3000-9892-11ea-8fe9-c58c86f39daa.png)